exercises/sieve/src/example.erl

```diff
@@ -0,0 +1,16 @@
+-module(example).
+
+-export([sieve/1, test_version/0]).
+
+sieve(N) when N < 2 ->
+    [];
+sieve(N) ->
+    sieve(lists:seq(2,N), []).
+
+sieve([], P) ->
+    lists:reverse(P);
+sieve(S, P) ->
```

Comment:
Can you please match `[N|S]` here?

Reply:
Happy to oblige.  It did occur to me at one point that I should not have to use hd/1 explicitly.

This GitHub UI is confusing me but no more than any 'intuitive' GUI does.  I am sure with time I will get the hang of it but if it has let me use it incorrectly do let me know.